libraries/AP_GPS/AP_GPS_SBF.h

```diff
@@ -185,4 +189,21 @@ class AP_GPS_SBF : public AP_GPS_Backend
     } sbf_msg;
 
     void log_ExtEventPVTGeodetic(const msg4007 &temp);
+
+    enum {
+        SOFTWARE      = (1 << 3),
+        WATCHDOG      = (1 << 4),
+        CONGESTION    = (1 << 6),
+        MISSEDEVENT   = (1 << 8),
+        CPUOVERLOAD   = (1 << 9),
+        INVALIDCONFIG = (1 << 10),
+        OUTOFGEOFENCE = (1 << 11),
+    } RxError_bits;
+
+    #define RX_ERROR_MASK (SOFTWARE      | \
```

Comment:
Sorry, should have asked this before, but can we put this in the cpp file? It reduces macro scope (which besides being safer, reduces the number of files needed to rebuild when this changes).